brainspell/search_helpers.py

```diff
@@ -49,7 +49,7 @@ def parse_helper(query):
         columns.append(Articles.pmid)
     if tiab.search(query):
         columns.extend([Articles.title, Articles.abstract])
-    formatted_query = re.sub('\[.*\]', '', query).strip().replace(" ", "%")
+    formatted_query = re.sub(r'\[.*\]', '', query).strip().replace(" ", "%")
```

Comment:
Did this not work the way it was before?

Reply:
Looks like this was a feature of autopep8, it now turns regex string arguments into raw strings.